src/main/java/it/gov/pagopa/reward/notification/service/feedback/RewardNotificationFeedbackMediatorServiceImpl.java

```diff
@@ -99,19 +102,32 @@ protected Mono<List<RewardOrganizationImport>> execute(List<StorageEventDto> pay
         return Flux.fromIterable(payload)
                 // consider just organization feedback upload events
                 .filter(this::isOrganizationFeedbackUploadEvent)
-                .doOnNext(i->log.info("[REWARD_NOTIFICATION_FEEDBACK] Processing import request: {}", i.getSubject()))
+                .doOnNext(i -> log.info("[REWARD_NOTIFICATION_FEEDBACK] Processing import request: {}", i.getSubject()))
                 .map(mapper)
                 // trace import request
                 .flatMap(importsRepository::createIfNotExistsOrReturnEmpty)
                 // retrieve and elaborate csv
                 .flatMap(this::retrieveAndElaborateCsv)
                 // finalize import request state and store it
                 .flatMap(this::finalizeImportRequest)
-                .doOnNext(i->log.info("[REWARD_NOTIFICATION_FEEDBACK] Import request processing results stored: {}", i.getFilePath()))
+                .flatMap(this::sendEmail)
+                .doOnNext(i -> log.info("[REWARD_NOTIFICATION_FEEDBACK] Import request processing results stored: {}", i.getFilePath()))
                 .collectList();
     }
 
+    private Mono<RewardOrganizationImport> sendEmail(RewardOrganizationImport i) {
+
+        return EmailNotificationConstants.IMPORT_ELABORATED_STATUS_LIST.contains(i.getStatus())
```

Comment:
I think that we should notify the user in any case, also if error...
So probably there will not be a filter based on the status...